express-post-json/server.js

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+
+const app = express();
+let nextId = 1;
+const grades = {};
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => res.json(Object.values(grades)));
+
+app.post('/api/grades', (req, res) => {
+  grades[nextId] = { id: nextId, ...req.body };
```

Comment:
I would do these in the reverse order. The reason is that the last one wins, so if `req.body` happens to have an `id` property in it, that will override the `nextId` value, which is not what you want.